Statistics documentation (#527)
inveniosoftware/docs-invenio-rdm: docs/develop/howtos/backup_search_indices.md

Author: still needs some love

Author: ![image](https://user-images.githubusercontent.com/6437519/229880370-68792dfb-30c8-4f7e-bd3e-12bfa05df18e.png)

# Back up search indices

Starting with InvenioRDM v12, not all search indices can be recreated from the database
anymore.
In order to not lose any data when something goes wrong with the search indices, they
should be backed up regularly.

## Operations

Since our only copy of stats is stored in the indices of Elasticsearch in case
of a cluster error or failure we will lose our stats data. Thus it is advised
to setup a backup/restore mechanism for projects in production.

> We have several options when it comes down to tooling and methods for preserving
> our data in Elasticsearch.

Reviewer: ```suggestion
There are several options when it comes down to tooling and methods for preserving
our data in Elasticsearch.
```

To avoid saying that we actually support and have used all of the below tools in InvenioRDM (in reality we've only used `elasticdump` in the past).
